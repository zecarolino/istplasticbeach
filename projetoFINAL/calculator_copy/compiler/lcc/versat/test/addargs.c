/* function cannot be named after any of the opcodes, 'add' for instance, or
   the assembeler will mess up */
int sum(int x, int y) { return x + y; }
int main() { return sum(5,4); }
