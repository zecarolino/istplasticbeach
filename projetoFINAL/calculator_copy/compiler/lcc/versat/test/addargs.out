VCD info: dumpfile xtop.vcd opened for output.
9
Catched Trap at time  483 clock cycles:
